fighthealthinsurance/fax_actor.py

```
@@ -5,11 +5,14 @@
 import time
 import asyncio
 
-from django.core.mail import EmailMultiAlternatives
+from django.core.mail import EmailMultiAlternatives, BadHeaderError
 from django.template.loader import render_to_string
 from django.utils import timezone
 from django.urls import reverse
 from fighthealthinsurance.utils import get_env_variable
+from smtplib import SMTPException
```

Ruff (0.8.2)

`smtplib.SMTPException` imported but unused

Remove unused import: `smtplib.SMTPException`

(F401)